python/grass/jupyter/utils.py

```diff
@@ -225,7 +226,14 @@ def save_vector(name, geo_json):
 
 
 def get_number_of_cores(requested, env=None):
-    """Get the number of cores to use for multiprocessing."""
+    """
+    Get the number of cores to use for multiprocessing.
```

Comment:
Just a small detail, but please format it this way (see [Sphinx docstring format](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html))
```suggestion
    """Get the number of cores to use for multiprocessing.
```